src/selectors/url-state.js

```diff
@@ -190,7 +190,7 @@ export const getProfileName: Selector<null | string> = createSelector(
       return null;
     }
     const pathParts = pathInZipFile.split('/');
-    return pathParts[pathParts.length - 1];
+    return pathParts[pathParts.length -2]+ '/' + pathParts[pathParts.length -1] ;
```

Comment:
This isn't enough, as you need to take care of the cases where there's only 1 part.

Also the CI is failing, I believe this is a linting problem, that you can fix by running `yarn lint-fix`.